### DISCUSSION TITLE
High resolution global simulations using the `HydrostaticFreeSurfaceModel`

### DISCUSSION THREAD

**Discussion by simone-silvestri** [CliMA/Oceananigans.jl#3207] [Show and tell]
Following the distributed optimization of Oceananigans, we have been performing global simulations using the `HydrostaticFreeSurfaceModel` and an `ImmersedBoundaryGrid` to represent bathymetry to benchmark the performance and scalability of our code base.

Although performance, and not results, was the focus of the investigation, these simulations still produced quite some mesmerizing results:

10 km resolution, temperature, and vorticity in the Pacific, the Kuroshio, and the Aghulas current

https://github.com/CliMA/Oceananigans.jl/assets/33547697/6445c9b7-1ffe-4f2a-b242-ccb31c4e0aa6

https://github.com/CliMA/Oceananigans.jl/assets/33547697/0cdbf4aa-3e89-484b-be6a-1c2cc762a617

https://github.com/CliMA/Oceananigans.jl/assets/33547697/e2b63e81-a937-4bd7-b255-0f1404c397a9

https://github.com/CliMA/Oceananigans.jl/assets/33547697/9b26b0f4-a6c9-4591-a53d-a837b3e3a8ec

**Comment by glwagner** [comment]
~~Are these movies or static images?~~ The movies work! Probably we should adapt some of these for the README!
